config/package/resources.yaml.gotmpl

```diff
@@ -162,7 +153,8 @@ webhooks:
     - CREATE
     - UPDATE
     resources:
-    - imagecontentpolicies
+    - imagedigestmirrorsets
```

Comment:
Just noting that this seems to be a consequence of PR #221 merging after PR #220 and having the old HS package contents in it. Which seems a problem that needs to be detected in future by `make` validation. I will create a follow-up ticket to track that.

But to summarize - this change is expected and fine.